chord_metadata_service/utils.py

```diff
@@ -16,3 +18,21 @@ async def build_id_set(qs: QuerySet, field: str) -> Set[Any]:
 
 async def build_id_set_from_model(m: Type[Model], field: str) -> Set[Any]:
     return await build_id_set(m.objects.all(), field)
+
+
+def build_experiments_by_subject(biosamples_experiments_details: List[Dict]) -> Dict[str, List[Dict]]:
```

Comment:
this isn't a very generic util - it should probably be kept in the `chord` package, or maybe in `restapi`

Reply:
it should also be in the same file as `get_biosamples_with_experiment_details`, so maybe that function should move to restapi/chord utils as well.